oidc-provider/.gitignore

```diff
@@ -1,4 +1,7 @@
 bin/
 lib/
+man/
 include/
-pip-selfcheck.json
+**/pip-selfcheck.json
```

Comment:
I think this pattern is functionally equivalent to the one that you deleted.  If so, could you revert this change?

http://git-scm.com/docs/gitignore